pkg/clients/aws.go

```diff
@@ -597,6 +597,21 @@ func LateInitializeIntFrom32Ptr(in *int, from *int32) *int {
 	return nil
 }
 
+// LateInitializeIntFromInt32Ptr returns in if it's non-nil, otherwise returns from
+// which is the backup for the cases in is nil.
+func LateInitializeIntFromInt32Ptr(in *int, from *int32) *int {
```

Comment:
Let's also mark `LateInitializeIntFrom32Ptr` as deprecated so that we can later remove it.

And also please comment on the difference between two versions.